src/UI/templates/default/Panel/tpl.report.html

```diff
@@ -1,4 +1,4 @@
-<div class="panel panel-primary">
+<div class="panel panel-primary il-panel-report">
```

Comment:
Thanks for adding il-panel-report. IMHO, this is exactly how we should use CSS classes. il-[name-of-family]-[name-of-component] on top level and all CSS followed from there. Nice.